.github/l10n/linter_config.yml

```diff
@@ -20,10 +20,8 @@ ID02:
 CO01:
   enabled: true
   brands:
-    - Fakespot
```

Comment:
Perhaps removing this should happen in a separate PR, since it's unrelated to Pocket, but it's probably not significant.